lib/client.js

```diff
@@ -14,7 +14,7 @@ function getValue (field, callback) {
         timeout: this.timeout,
         json: true
     }, function (error, response, body) {
-        if (!error && response.statusCode === 200 && body.value) {
+        if (!error && response.statusCode === 200 && (body.value || body.value === "")) {
```

Comment:
Didn't realize this, doesn't this mean `0` and `false` are also going to fail?  Can we change this to `body.value !== undefined`